server/activate_hooks.go

```diff
@@ -56,6 +59,17 @@ func (p *Plugin) OnActivate() error {
 		}
 	}
 
+	job, cronErr := cluster.Schedule(
```

Comment:
I quite like the API. :+1: 

I'm wondering if the makes sense to have the `API` as the first parameter, similar how `ctx` should always be the first argument. It makes sense to me to put the parameters that are most of the time the same as the first ones.

Reply:
Agreed!